### ISSUE TITLE
Question - Is it Possible to Pass Additional Info to the EMR Cluster using Terraform

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#1178]
_This issue was originally opened by @roman1627 as hashicorp/terraform#15570. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

We are trying to create an AWS EMR Cluster using Terraform (v0.9.11) and wanted to know if there is a way to Pass Additional Info like below via Terraform -

```
{ 'ami64':'ami-XXXXX', 'amiHvm64': 'ami-XXXXXXX', 'customAMI': true, 'hadoopConfigurationVersion': '4.0', 'instanceAwsClientConfiguration': { 'proxyPort': 8099, 'proxyHost': 'XXXXXXXX' } }
```

Could you please help us with above query?

**Comment by DigitalPig** [comment]
I would like to echo @roman1627 's question here. I did try to use `configurations` field but it does not work. Apparently it looks like there must be a parameter in tf aws provider to supply `additional_info` field to get this to work. 

Also, custom AMI is really important for us and I did try out PR #2766 and it works great on our side. Really wish it could be incorporated into the main repo as soon as possible. (Thanks @gordonbondon)

**Comment by Ninir** [comment]
Hi folks, 

this has been merged and will be released in 1.8.0.

Tanks for your patience and Happy Terraforming!

**Comment by bflad** [comment]
The `custom_ami` support has been released in [terraform-provider-aws version 1.8.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#180-january-29-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
